pkg/reconciler/route/resources/certificate.go

```diff
@@ -58,6 +61,9 @@ func MakeCertificates(route *v1alpha1.Route, domainTagMap map[string]string) []*
 				Name:            certName,
 				Namespace:       route.Namespace,
 				OwnerReferences: []metav1.OwnerReference{*kmeta.NewControllerRef(route)},
+				Annotations: resources.UnionMaps(map[string]string{
```

Comment:
I'd revert the order in the union maps, since the second one wins. And you want your annotation to win.

Reply:
Done